addons/website/static/src/js/content/menu.js

```diff
@@ -678,12 +698,24 @@ publicWidget.registry.HeaderMainCollapse = publicWidget.Widget.extend({
      */
     _onCollapseShow() {
         this.el.classList.add('o_top_menu_collapse_shown');
+        if (this.offcanvasAndBoxedHeader) {
+            this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive();
+            this.callToActionEl.classList.add('nav-item');
+            this.navbarEl.append(this.callToActionEl);
```

Comment:
Maybe you should check if `this.callToActionEl` exists, since it could not be present if someone removed it from the template (as a customization for example) :thinking: (same in `_onCollapseHidden`)